### ISSUE TITLE
Input has no label

### ISSUE THREAD

**Issue by wouterBouwmeester** [JedWatson/react-select#3778]
Im getting a warning in my storybook that the react-select component does not confirm to WCAG.

The input inside react-select__input (react-select-4-input) does not have a label.
![image](https://user-images.githubusercontent.com/24525299/65664280-4d879680-e039-11e9-96d2-3e7faab1ee15.png)

**Comment by malufett** [comment]
up

**Comment by razh** [comment]
You need to add the `<label>` element yourself.

```jsx
<>
  <label htmlFor="example">Example label</label>
  <Select inputId="example" />
</>
```

**Comment by wouterBouwmeester** [comment]
That works perfect @razh . Thanks
